Command/SshCommand.php

```diff
@@ -99,6 +99,15 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         $slices = array_unique(array_values($this->client->getDbList()));
         $processArgs = ['ssh', '-N'];
+
+        if ($output->isVerbose()) {
```

Comment:
I'm not sure why, but this appears to always be true regardless of whether `-v` is passed. It is false if `-q` is passed, but without either of those flags the output shouldn't be verbose, I think.

Reply:
Definitely odd; this doesn't seem to be the case for me. `-v` is only explicitly set if I ran the command with `-v`, and [`isVerbose()`](https://symfony.com/doc/current/console/verbosity.html) should only be true when that argument is passed as well. Perhaps the `SHELL_VERBOSITY` environment variable was inadvertently set to `1`?

Reply:
Ah, it looks like it's only when `APP_ENV=dev`, which does make sense. Does it do that for you too? I'm also testing with WS Export.

Reply:
I was testing with CopyPatrol but after testing with WS Export, yes, that seems to be the case. For some reason, this behavior doesn't happen on CopyPatrol (even with the same `config/packages/dev/monolog.yaml` and `APP_ENV=dev`).

Reply:
Well, it's certainly a separate issue. The code here all looks good, let's merge it.